core/lib/domain/extensions/datetime_extension.dart

```diff
@@ -21,6 +21,13 @@ extension DateTimeExtension on DateTime {
     final to = DateTime(year, month, day);
     return (to.difference(from).inHours / 24).round().abs();
   }
+
+
+  int minutesBetween(DateTime from) {
+    from = DateTime(from.year, from.month, from.day);
+    final to = DateTime(year, month, day);
```

Comment:
to compare minute, DateTime need have minute
```suggestion
    from = DateTime(from.year, from.month, from.day, from.hour, from.minute, from.second);
    final to = DateTime(year, month, day, hour, minute, second);
```